.github/workflows/publish_release_candidate.yml

```
@@ -0,0 +1,72 @@
+    secrets: inherit
+
+  create_release:
+    name: Create GitHub release
+    needs:
+      - backend_release
+      - frontend_release
+      - prepare
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)